### PR TITLE
fix(spacewidget): fix reflow of content in 320px wide browsers

### PR DESCRIPTION
**Pull request by bhabalan**
### Problem
Currently, in the react space widget, when the browser width is at 320px, alot of items get cropped on screen.
This is an issue for users on smaller screens as they cannot access all the content and controls.

### Fix
CSS fixes have been added to the activity component and its children for 320px so that all the content and controls are visible and available to the user

JIRA: [SPARK-549499](https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-549499)

Vidcast: https://app.vidcast.io/share/7c9cb830-9d15-44e9-87c5-384c5ed224d3

Before:
image

After:
image

### CODE REVIEW

File: packages/node_modules/@webex/react-component-activity-share-files/src/index.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ import AdaptiveCard from '@webex/react-component-adaptive-card';
 import {CARD_CONTAINS_IMAGE} from '@webex/react-component-utils';
 
 import styles from './styles.css';
+import './momentum.scss';
```
Comment (mkesavan13) [inline packages/node_modules/@webex/react-component-activity-share-files/src/index.js:12]: So, until we import here, the momentum overrides won't take effect. But after this, the CSS is going to stay in the `CSSOM` right?

Won't that override the CSS affecting the application where the Widget would reside? (Assuming that app also uses momentum library)

File: packages/node_modules/@webex/react-component-activity-share-files/src/styles.css [hunk 2]
```diff
@@ -4,7 +4,7 @@
 }
 
 .shareList {
-  width: 320px;
+  max-width: 320px;
```
Comment (mkesavan13) [inline packages/node_modules/@webex/react-component-activity-share-files/src/styles.css:7]: Won't it feel inconsistent if the width of shared file A and shared file B is different based on the length of their names?

File: packages/node_modules/@webex/react-component-activity-text/src/styles.css [hunk 3]
```diff
@@ -11,6 +12,14 @@
   transition: color 0.3s, margin-left 0.3s ease 0.5s;
 }
 
+@media (max-width: 320px) {
+  /* Reply text shouldn't get cropped */
```
Comment (mkesavan13) [inline packages/node_modules/@webex/react-component-activity-text/src/styles.css:16]: In which case shouldn't we having a word-wrap here?

**Review by mkesavan13 — CHANGES_REQUESTED** [review]
Mostly looks good. Only a few comments.

File: .gitattributes [hunk 4]
```diff
@@ -1,2 +1,3 @@
 /node_modules linguist-vendored
 /packages/node_modules linguist-vendored=false
+/packages/node_modules/** linguist-generated=false
```
Comment (mkesavan13) [inline .gitattributes:3]: Thanks for that 😎

File: packages/node_modules/@webex/react-component-activity-item-base/src/styles.css [hunk 5]
```diff
@@ -100,6 +110,15 @@
   margin-left: 56px;
 }
 
+@media (max-width: 320px) {
+.activityItem.additional {
+  /*
+   * reduce wasted space for smaller screens
+   */
+  margin-left: 48px;
+}
```
Comment (mkesavan13) [inline packages/node_modules/@webex/react-component-activity-item-base/src/styles.css:119]: nitpick: Could you please indent this?

File: packages/node_modules/@webex/react-component-activity-share-files/src/styles.css [hunk 6]
```diff
@@ -7,6 +7,65 @@
...
+    max-width: 100%;
+    width: unset;
+  }
+
+  .shareList div[class*="md-content__chat-inner-container"] {
+    width: unset;
+    min-width: 170px;
+  }
+
+  .shareList div[class=" md-content-file--full"] {
```
Comment (mkesavan13) [inline packages/node_modules/@webex/react-component-activity-share-files/src/styles.css:21]: Why do we have a space here?
```suggestion
  .shareList div[class="md-content-file--full"] {
```

**Review by mkesavan13 — APPROVED** [review]
A couple of nitpick comments. Looks good otherwise.

**Approving**

File: packages/node_modules/@webex/react-component-activity-share-files/src/styles.css (hunk 6 above)
Comment (bhabalan) [reply packages/node_modules/@webex/react-component-activity-share-files/src/styles.css:21]: It's an exact match only if there is a preceding space
image

**Review by Shreyas281299 — APPROVED** [review]

**Pull request merged**